layers/+misc/multiple-cursors/funcs.el

```diff
@@ -0,0 +1,34 @@
+;; -*- lexical-binding: t -*-
+;;
+;;; funcs.el --- Spacemacs Multiple Cursors Layer packages File
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Codruț Constantin Gușoi <codrut.gusoi@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defun spacemacs//paste-transient-state-p ()
```

Comment:
I would rename this function to `spacemacs//evil-mc-paste-transient-state-p`. The current name is too broad.